phpunit.xml.dist

```diff
@@ -21,6 +22,8 @@
     </testsuites>
 
     <listeners>
+        <listener class="Symfony\Bridge\PhpUnit\SymfonyTestsListener" />
```

Comment:
What does this do? Thx

Reply:
Don't know exactly ... but the official recipe includes it: https://github.com/symfony/recipes/blob/3b19bd94886555df1997b441b6cf6a1fab088f3a/symfony/phpunit-bridge/4.1/phpunit.xml.dist#L31